PSSwagger.LiveTestFramework/build/PSSwagger.LiveTestFramework.Build.psm1

```diff
@@ -0,0 +1,338 @@
+Microsoft.PowerShell.Core\Set-StrictMode -Version Latest
+
+<#
+.DESCRIPTION
+  Ensures all dependencies required for running tests are present on the current machine.
+#>
+function Initialize-Dependencies {
+    [CmdletBinding()]
+    param()
+
+    $expectedDotNetVersion = "2.1.0-preview1-006547"
+    Write-Host "Setting up PSSwagger.LiveTestFramework.Build dependencies:"
+    Write-Host "    dotnet: $expectedDotnetVersion"
+    Write-Host "    PSSwaggerUtility: *"
+    Write-Host ""
+    $failed = Setup-PSSwaggerUtility
+    if (-not $failed) {
+        Initialize-PSSwaggerDependencies
+        Import-Module -Name PSSwaggerUtility
+        $r = Setup-DotNet | Select-Object -Last 1
+        $failed = $failed -or $r
+    }
+
+    if ($failed) {
+        Write-Error -Message 'One or more dependencies failed to intialize.'
+    } else {
+        Write-Host "Completed setting up PSSwagger.LiveTestFramework.Build dependencies." -BackgroundColor DarkGreen
+    }
+
+    $failed
+}
+
+<#
+.DESCRIPTION
+  Build PSSwagger LiveTestFramework Console Server and libraries using dotnet CLI.
+  Optionally copies binaries to OutputDirectory parameter value if available.
+#>
+function Invoke-Build {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$false)]
+        [string]
+        [ValidateSet('net452')]
+        $Framework = 'net452',
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        [ValidateSet('Debug','Release')]
+        $Configuration = 'Debug',
+
+        [Parameter(Mandatory=$false)]
+        [string]
+        $OutputDirectory,
+
+        [Parameter(Mandatory=$false)]
+        [switch]
+        $CleanOutputDirectory
+    )
+
+    Initialize-Dependencies
+    if ($OutputDirectory -and $CleanOutputDirectory -and (Test-Path -Path $OutputDirectory -PathType Container)) {
+        Remove-Item -Path $OutputDirectory -Recurse
+    }
+
+    if ($OutputDirectory -and (-not (Test-Path -Path $OutputDirectory -PathType Container))) {
+        $null = New-Item -Path $OutputDirectory -ItemType Directory
+    }
+
+    Get-ChildItem -Path (Join-Path -Path $PSScriptRoot -ChildPath ".." | Join-Path -ChildPath "src" | Join-Path -ChildPath "*.csproj") -File -Recurse | ForEach-Object {
```

Comment:
>Join-Path -Path $PSScriptRoot -ChildPath ".." [](start = 25, length = 45)

Consider using below for getting the parent folder path.

Split-Path -Path $PSScriptRoot -Parent